### DISCUSSION TITLE
Sponcer's Profile isn't getting verified.

### DISCUSSION THREAD

**Discussion by AbhinavVihan** [community/community#39149] [Other Feature Feedback, Questions, & Ideas] [answered]
Question

### Body

My Github Sponcer's Profile isn't getting verified, says (***Your Profile isn't live yet, We need More information to continue***).
Can anyone assist a little please :)

**Comment by DSDmark** [comment]
Fill all details which **GitHub** ask.
[Sponsors dash](https://github.com/sponsors/DSDmark/dashboard).

Here are all details how to proceed.
[**GitHub Docs**](https://docs.github.com/en/sponsors/receiving-sponsorships-through-github-sponsors/setting-up-github-sponsors-for-your-personal-account)

**Reply by AbhinavVihan** [reply]
@DSDmark  Thankyou for this
But I've filled all the information mentioned here.

**Reply by DSDmark** [reply]
@AbhinavVihan,
So, According to the **GitHub Documentation**.
> If you have a bank account in a supported region, GitHub will review your application within two weeks.

So, I guess you need to wait ⏲️.

**Reply by DSDmark** [reply]
Hi @AbhinavVihan,

😊 I hope you get your answer. If you get your response, you should close that question by clicking ✅ accept answer button, or you can ✍️ write the answer and accept your own answer.

> **Note**: if you close this question, then nobody will contribute to your question anymore. So, close this question when you satisfy your answer, or you can write your own question with a better explanation

**Comment by cheshire137** [comment, marked as answer]
Hi there! I'm a GitHub developer on the Sponsors team. I just checked, looks like your Sponsors profile got approved. 👍🏻 https://github.com/sponsors/AbhinavVihan

**Reply by AbhinavVihan** [reply]
Thank You all for your help.

**Reply by ghost** [reply]
I also have same problem please do something it will mean a lot
